includes/DataObjects/RequestForm.php

```diff
@@ -19,23 +21,98 @@ class RequestForm extends DataObject
     /** @var int */
     private $domain;
     /** @var string */
-    private $name;
+    private $name = '';
     /** @var string */
-    private $publicendpoint;
+    private $publicendpoint = '';
     /** @var string */
-    private $formcontent;
+    private $formcontent = '';
```

Comment:
Because PHP.

Actually, because the `get*()` methods in RequestForm are type-restricted to be a string, and PHP defaults variables to null. Since these are required fields, I don't really *want* to make them nullable (the `string?` type annotation)

For both creation and edits, `PageRequestFormManagement` calls the `populateFromObject` method to make it easier to set all the field assignments on the template, but for new objects it's done via `populateFromObject(new RequestForm())`, thus all the fields are default values, and (without this change) the fields are all null which doesn't match the type annotation on the `get*()` methods. For a recent example of this sort of thing happening in prod, see #754